.azure-pipelines.yml

```diff
@@ -28,25 +28,32 @@ jobs:
   - script: |
       python -m pip install --upgrade pip
       pip install pytest-azurepipelines wheel
-      pip install -e .[dev,doc,test,louvain,leiden,magic,scvi,harmony]
+      pip install .[dev,doc,test,louvain,leiden,magic,scvi,harmony]
     displayName: 'Install dependencies'
 
-  - script: |
-      pip install git+https://github.com/theislab/anndata
+  - script: pip install git+https://github.com/theislab/anndata
     displayName: 'Install development anndata'
     condition: eq(variables['ANNDATA_DEV'], 'yes')
 
+  - script: pytest --ignore=tests/_images
+    displayName: 'PyTest'
+
+- job: static_analysis
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.8'
+    displayName: 'Use Python 3.8'
+
+  - script: pip install black docutils
+    displayName: 'Install dependencies'
```

Comment:
I'm not sure this should be a separate job. I would prefer we use the jobs we have for other operating systems. Azure allows us to have multiple checks within a job, so I'm not sure it's that useful to put static analysis in it's own build.

Reply:
that way it runs much faster, so people see earlier if they didn’t format things properly